README.md

```diff
@@ -8,6 +8,7 @@
 1. Make a copy of `.env.sample` to `.env` and change the values of variables with original values.
 1. Migrate the migrations: `python manage.py migrate`
 1. Run the app: `python manage.py runserver`
+1. Open two terminals and run `celery -A server worker -l info` and `redis-server`  on second and third terminal respectively.
```

Comment:
A brief of how to install redis will be helpful. I will put that as a good first issue for now. :)